zipkin/src/main/java/zipkin/internal/ThriftCodec.java

```diff
@@ -110,6 +110,7 @@ public Span readSpan(byte[] bytes) {
     final Field IPV4 = new Field(TYPE_I32, 1);
     final Field PORT = new Field(TYPE_I16, 2);
     final Field SERVICE_NAME = new Field(TYPE_STRING, 3);
+    final Field IPV6 = new Field(TYPE_STRING, 4);
```

Comment:
First, I'll own any bug on this addition.

To answer your question, there aren't compiler (libthrift, scrooge, etc)
interop tests for thrift (or openapi interop tests for json) in this repo.

The thrifts live in a separate repo
<https://github.com/openzipkin/zipkin-api/tree/master/thrift>, and making
such tests would add maintenance due to factors including the race
condition of adding new fields like this. Such a thing would also be a
heavier build as we'd need to choose which generator (assuming apache maybe
0.93?), hook it up for the test source (or make a module ignored from
publishing etc), then design some tests that pass data through. That's
guaranteed to be a lot more to maintain in this repo than exists now.

Since we add fields very rarely, I'm not sure I want to add all of that
right now. Regardless, it would be a large enough change to warrant a
separate pull request.